package.json

```diff
@@ -2,12 +2,13 @@
 	"name": "kolorist",
 	"version": "1.7.0",
 	"description": "A tiny utility to colorize stdin/stdout",
-	"main": "dist/cjs/index.js",
-	"module": "dist/module/index.js",
-	"types": "dist/types/index.d.ts",
+	"main": "dist/index.cjs",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
 	"scripts": {
 		"test": "mocha -r @esbuild-kit/cjs-loader --extension ts,js src/*.test.ts",
-		"build": "rimraf dist/ && tsc && tsc -p tsconfig.module.json && tsc -p tsconfig.esm.json && node tools/post-build.js",
+		"type-check": "tsc --noEmit",
```

Comment:
Since we're no longer using `tsc` to build, this is just a convenience to use `tsc` only for type-checking during development. Can be removed if you rely on the IDE for type-checking.

For building, `npm run build` still type-checks because it uses TypeScript to generate the `.d.ts` file.